source/classes.tex

```diff
@@ -4782,9 +4767,8 @@
 
 \pnum
 \indextext{declaration!overloaded name and \tcode{friend}}%
-When a friend
-declaration refers to an overloaded name or operator, only the function specified
-by the parameter types becomes a friend.
+\begin{note}
+A friend declaration refers to an entity, not (all overloads) of a name.
```

Comment:
"(all overloads) of" -> "(all overloads of)"

Reply:
fixed